plugins/humanitec-backend-scaffolder-module/package.json

```diff
@@ -22,13 +24,13 @@
     "postpack": "backstage-cli package postpack"
   },
   "dependencies": {
-    "@backstage/backend-plugin-api": "^0.6.17",
-    "@backstage/plugin-scaffolder-node": "^0.3.3",
+    "@backstage/backend-plugin-api": "0.6.18",
```

Comment:
Locking this is required due to https://github.com/janus-idp/backstage-plugins/issues/1843